couch/parsers.py

```diff
@@ -24,14 +24,33 @@ def parse_couch_logs(logger, line):
         logger.exception('Failed to parse log line')
         return None
 
-    return ('couch.timings', timestamp, request_seconds, {
+    return [
+        get_couch_timing_gauge(timestamp, url, database, http_method, status_code, couch_url, request_seconds),
+        get_couch_requests_counter(timestamp, url, database, http_method, status_code, couch_url, request_seconds)
+    ]
```

Comment:
Nice I didn't realize you can return a list here. (Maybe we do that elsewhere and I just forgot.)

Reply:
we also do this with the nginx metrics